### PR TITLE
Add new keyboard "40rmie"

### PR DESCRIPTION
**Pull request by KuroTheCrazy**
## Description

The 40rmie, pronounced "forty-or-me" or "For-me" is a 40%ish keyboard designed by KuroTheCrazy. This board features base kit compatibility with a half numrow, partial fn row, nav cluster, and arrow keys, for perfect ease of use in gaming and work. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
Please review the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and verify that you have followed the checklist for the type of PR this is.

Additionally, since it looks like you used kbfirmware to generate the files, you may want to check out
https://noroadsleft.github.io/kbf_qmk_converter/

File: keyboards/40rmie/rules.mk [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE ?= yes	# Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE ?= yes	# Mouse keys(+4700)
+EXTRAKEY_ENABLE ?= yes	# Audio control and System control(+450)
+CONSOLE_ENABLE ?= no	# Console for debug(+400)
+COMMAND_ENABLE ?= no    # Commands for debug and configuration
+SLEEP_LED_ENABLE ?= no  # Breathing sleep LED during USB suspend
+NKRO_ENABLE ?= yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```
Comment (Jrodna) [inline keyboards/40rmie/rules.mk:53]:
```suggestion
NKRO_ENABLE ?= yes	# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```

**Review by Jrodna — CHANGES_REQUESTED** [review]
Just want to tidy this little bit

**Comment by drashna** [comment]
Honestly, it's been 5 months since this was opened and changes requested and zero updates have occurred.   Unlikely that any update will actually happen.  Closing.

**Pull request closed without merging**